### DISCUSSION TITLE
What are h and c in the inputs?

### DISCUSSION THREAD

**Discussion by goudbor** [snakers4/silero-vad#339] [Q&A] [answered]
Hey!  I'm trying to do inference directly in the browser using silero_vad.onnx , but not sure what the two inputs h and c are?  

Edit: I realized it was easier to just use the npm modules provided by @ricky0123 as it works for my use case, but would still like to know in general if somebody comes across this question, as I can't seem to find a paper for the silero_vad model.

**Comment by snakers4** [comment, marked as answer]
Hi, you can look up how these are used in the vad utils. There is a function called onnx wrapper or similar.
